support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -255,36 +255,46 @@
 </div>
 
 <div id="localization" class="page"
-  style="padding-top: 32px; padding-bottom: 32px; padding-left: 24px; padding-right: 24px;">
-  <div id="upload">
-    <div
-      style="color: black; font-size: 18; word-wrap: break-word">
-      Upload an existing gen_strings.xml that is generated by this plugin?
-    </div>
-
-    <div class="container">
-      <button class="styled-button" style="margin-top: 16px;">
-        <label for="stringsXmlInputFile">Browse</label>
-        <input type="file" id="stringsXmlInputFile" name="strings_xml_input_file" accept=".xml" class="hidden" />
-      </button>
+  style="padding-top: 16px; padding-bottom: 16px; padding-left: 24px; padding-right: 24px;">
+  <div id="optionsPanel">
+    <div id="localizationOptions">
+      <div style="border-bottom: solid #000000; font-weight: bold;">Exclude options:</div>
+      <div style="margin-top: 8px; margin-left: 16px;">
+        <input type="checkbox" id="excludeHashTagName" name="excludeOptions" value="excludeHashTagName" checked>
+        <label for="excludeHashTagName">Exclude text nodes whose name starts with '#'</label>
+      </div>
+      <div style="margin-top: 8px; margin-left: 16px;">
+        <input type="checkbox" id="excludeCustomization" name="excludeOptions" value="excludeCustomization" checked>
+        <label for="excludeCustomization">Exclude text nodes customized by client</label>
+      </div>
     </div>
-
-    <div id="inputStrings" style="display: none; margin-top: 16px;"></div>
-
-    <div class="container">
-      <button id="showOutputs" class="styled-button" style="margin-top: 16px;"
-        onclick="showOutputStrings()">Skip</button>
+    <div style="color: black; word-wrap: break-word; margin-top: 16px;">
+      <div style="border-bottom: solid #000000; font-weight: bold;">Upload options:</div>
+      <div style="margin-top: 8px; margin-left: 16px;"">
+        <input type="checkbox" id="uploadStringsXmlOption" name="uploadStringsXmlOption" checked />
+        <label for="uploadStringsXmlOption">Upload an existing gen_strings.xml that is generated by this plugin?</label>
+      </div>
+      <div id="stringInputPanel" style="margin-left: 18px;">
+        <div class="container" style="margin-top: 8px; margin-bottom: 8px;">
+          <button class="styled-button">
+            <label for="stringsXmlInputFile" style="width: inherit; height: inherit; line-height: 36px;">Browse</label>
+            <input type="file" id="stringsXmlInputFile" name="strings_xml_input_file" accept=".xml" class="hidden" />
+          </button>
+        </div>
+        <div id="inputStringsTableContainer" class="table-container" style="display: none; max-height: 320px;">
```

Comment:
Is this the embedded table inside the UI? It looks a little odd to me to have a table with horizontal and vertical scrollbars inside the main UI, which also has a vertical scrollbar. Is it possible to not have the inner table and make the main UI resizeable?

Reply:
I removed height limit of the table so it won't have a vertical scrollbar. But I don't know how to make the main UI resizable that it seems figma's api ShowUIOptions doesn't have a way to make it resizable.